doc/index.md

```diff
@@ -101,9 +101,8 @@ Consider:
 Notice that `meta::quote<std::add_pointer_t>` and `meta::quote_trait<std::add_pointer>`
 mean the same thing.
 
-\note You may wonder what advantage Callables have over alias templates. A Callable is a
-*type* that represents a computation. Much of Meta revolves around types and the
-computation of types. Sometimes it's desirable to compute a computation, or to use a
+\note You may wonder what advantage Callables have over alias templates. A Callable is a *type* that represents a computation. It is the metaprogramming 
+analogue to a functor class which has a call operator `operator()(...)`. Much of Meta revolves around types and the computation of types. Sometimes it's desirable to compute a computation, or to use a
```

Comment:
Please don't use the term "functor" unless you are talking about [Functor](https://en.m.wikipedia.org/wiki/Functor).

Reply:
"It is the metaprogramming analogue to a function object."